tests/default_vars.sh

```diff
@@ -490,12 +504,27 @@ export DO_MYNNSFCLAY=.false.
 export BL_MYNN_TKEADVECT=.false.
 
 # LSM
-export LSM=1
+export LSM=2
 export LSOIL_LSM=4
-export LANDICE=.true.
+export LANDICE=.false.
 export KICE=2
-export IALB=1
-export IEMS=1
+export IALB=2
+export IEMS=2
+export IOPT_DVEG=4
+export IOPT_CRS=2
+export IOPT_RAD=3
+export IOPT_ALB=1
+export IOPT_STC=3
+# P8
```

Comment:
Do we leave or remove these comments about P7, P8 etc?

Reply:
I say remove since part of the PR is to make all tests compatible with p8. If someone wanted to find older values/entries, then they could look at the commit history.

Reply:
Yes, that works for me.